server-spi/src/main/java/org/keycloak/credential/hash/PasswordHashProvider.java

```diff
@@ -30,6 +30,10 @@ public interface PasswordHashProvider extends Provider {
 
     PasswordCredentialModel encodedCredential(String rawPassword, int iterations);
 
+    /**
+     * Exists due the backwards compatibility. It is recommended to use {@link #encodedCredential(String, int)}
```

Comment:
Should we log an issue to remove this deprecated method at some point in the future? It should probably be attached to a specific Keycloak milestone.

Also, might be worth it to document this deprecation in the release notes/migration guide.

Reply:
Looks like the PR is merged, if you can log an issue @mposolda we can get this resolved.